### DISCUSSION TITLE
Welcome to Flight-Booking-System Discussions!

### DISCUSSION THREAD

**Discussion by nileshkapse** [nileshkapse/Flight-Booking-Reservation-System#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ultragem** [comment]
Hello.....please how do o connect the backend to the front end?
And should they be in different folders?

**Comment by nileshkapse** [comment]
https://github.com/VIIT-Project-Group/Flight-Booking-Reservation-System-Backend this is backend folder
